clients/src/test/java/org/apache/kafka/clients/consumer/internals/CommitRequestManagerTest.java

```diff
@@ -353,7 +380,8 @@ private void testRetriable(final CommitRequestManager commitRequestManger,
                                final List<CompletableFuture<Map<TopicPartition, OffsetAndMetadata>>> futures) {
         futures.forEach(f -> assertFalse(f.isDone()));
 
-        time.sleep(500);
+        // The manager should backoff for 100ms
+        time.sleep(100);
```

Comment:
changed to 100 because we are now using jitter = 0, so the next backoff should always be 100ms